### PR TITLE
Add UI for host page

### PR DESCRIPTION
**Pull request by dmfilipenko**
Screenshot 2022-03-10 at 21 15 42

1. Didn't figure out how to add translation to select length of stay

2. Commented api call

3. Added date react datapicker + custom css, which is making it quite similar in style to chakra ui

### CODE REVIEW

File: frontend/src/components/HostSignupForm.tsx [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+          </FormLabel>
+          <Select
+            id='region'
+            {...register('summary.availability.length_of_stay')}
+          >
+            <option key='Less than 3 days' value='Less than 3 days'>
+              {i18n._('less_3_days')}
+            </option>
+            <option key='One week or less' value='One week or less'>
+              {i18n._('one_week_less')}
```
Comment (rmadsen) [inline frontend/src/components/HostSignupForm.tsx:237]: Here I think you just do:
```jsx
<option ...> 
   <Trans>One week or less</Trans>
</option>
```

File: frontend/src/components/HostSignupForm.tsx [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+          </FormLabel>
+          <DatePicker
+            id='avaliability-date'
+            selected={parseDate(dateAvailable.field.value)}
+            onChange={(date) =>
+              dateAvailable.field.onChange(formatDate(date as Date))
+            }
+            disabled={!getValues('summary.availability.active')}
+            showPopperArrow={true}
+            dateFormat='dd-MM-yyyy'
```
Comment (rmadsen) [inline frontend/src/components/HostSignupForm.tsx:222]: Just to make sure -- the API format is YYYY-MM-DD (I think). Can you make sure that's what we send back to the server?

**Review by rmadsen — APPROVED** [review]
Nice!

File: frontend/src/components/HostSignupForm.tsx (hunk 1 above)
Comment (dmfilipenko) [reply frontend/src/components/HostSignupForm.tsx:237]: It doesnt work, it shows `[object Object]`, it's a known issue. So, somewhere I've read we need to use like this ` {i18n._('one_week_less')}` but as far I understood, all key translations is auto-generates, we cant add it manually?

**Pull request merged**